src/static/styles/partials/_base.scss

```diff
@@ -130,7 +130,6 @@ figcaption {
   color: $text-grey-color;
 
   a {
-    text-decoration: none;
```

Comment:
With `text-decoration: none` set, you couldn't distinguish the `<a>` links in the figure captions.